rtl/eclrtl/rtlnewkey.hpp

```diff
@@ -0,0 +1,76 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#ifndef RTLNEWKEY_INCL
+#define RTLNEwKEY_INCL
+#include "eclrtl.hpp"
+
+enum TransitionMask : byte
+{
+    CMPlt = 0x01,
+    CMPeq = 0x02,
+    CMPgt = 0x04,
+    CMPmin = 0x08,      //minimum possible value
+    CMPmax = 0x10,      //maximum possible value
+    CMPle = CMPlt | CMPeq,
+    CMPge = CMPgt | CMPeq,
+    CMPminmask = CMPgt|CMPmin,
+    CMPmaxmask = CMPlt|CMPmax,
+};
+inline constexpr TransitionMask operator | (TransitionMask l, TransitionMask r) { return (TransitionMask)((unsigned)l | (unsigned)r); } \
```

Comment:
Spurious trailing \

Reply:
I'll move something from jstats and reuse that